src/test/java/core/basesyntax/RegistrationValidationTest.java

```diff
@@ -0,0 +1,106 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationValidationTest {
+    private static RegistrationService registrationService;
+    private User testUser;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        testUser = new User();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_existingUser_notOk() {
+        validUser();
```

Comment:
Look for another implementation
```suggestion
        User user = new User(...
        Storage.PEOPLE.add(user);
        User newUser = new User(....
        assertThrows(RegistrationException.class, () -> registrationService.register(newUser));
```